src/ui/localization/locale/es.json

```
@@ -0,0 +1,56 @@
+{
+  "periods.week": "{{amount}} semana",
+  "periods.month": "{{amount}} mes",
+  "periods.year": "{{amount}} año",
+  "periods.day": "{{amount}} día",
+  "periods.weekShort": "{{amount}}wk",
+  "periods.monthShort": "{{amount}}mo",
+  "periods.yearShort": "{{amount}}yr",
+  "periods.dayShort": "{{amount}}d",
```

Hmm these short periods weren't translated to spanish it seems... Not sure if this is the best translation since it's pretty close to the non-short version but a suggestion:
```suggestion
  "periods.weekShort": "{{amount}}/sem",
  "periods.monthShort": "{{amount}}/mes",
  "periods.yearShort": "{{amount}}/año",
  "periods.dayShort": "{{amount}}/día",
```

---

Given the answer from above I will take the suggestion removing the `/`